tests-node/htmlbars-plugin-test.js

```diff
@@ -0,0 +1,33 @@
+var test = require('tape');
+var htmlbars = require('htmlbars/dist/cjs/htmlbars-syntax');
+var ClassTransformPlugin = require('../lib/htmlbars-plugin');
+
+testTransformation('creating a class attribute', {
+  input: 'local-class="foo"',
+  statementOutput: 'class=(concat styles.foo)', // FIXME superfluous concat
+  elementOutput: 'class="{{styles.foo}}"'
+});
+
+testTransformation('appending to a class attribute', {
+  input: 'class="x" local-class="foo"',
+  statementOutput: 'class=(concat "x" " " styles.foo)',
+  elementOutput: 'class="x {{styles.foo}}"'
+});
```

Comment:
It should be fine, for the more specific tests we can create other helpers or just be more verbose/explicit in those tests